ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/CliSandboxOnXRunner.scala

```diff
@@ -9,6 +9,13 @@ import com.daml.logging.ContextualizedLogger
 import com.daml.resources.ProgramResource
 
 object CliSandboxOnXRunner {
+  // TODO ED: Remove flag once explicit disclosure is deemed stable and all
+  //          backing ledgers implement proper validation against malicious clients
+  //
+  // NOTE: The flag is explicitly extracted out of the provided program arguments
+  //       and not passed via the HOCON config in order to prevent accidental
+  //       enablement, which could render participants vulnerable to malicious clients.
```

Comment:
I dont buy this argument that we should be doing this to prevent accidental enablement. We could accidentaly do a lot of bad stuff with current HOCON configuration, even making it not functional, still we do it through single configuration layer :) 

I would just have it as config option within HOCON.

Reply:
Maybe the message is not grave enough here: we would allow clients to bypass validation and successfully submit malformed contracts without anyone realising. We can't allow this

Reply:
Still I would be using config layer, but I do not see it as very big concern as this stuff will soon be gone :)

Reply:
Tudor is right about this. ED will be in this limbo state for a while. Only once we implement this across the stack, will it be safe to add it to HOCON